From dbdee8a7b1b8c7638314b96aa13c3156efe0afe3 Mon Sep 17 00:00:00 2001
From: Hauke Mehrtens <hauke@hauke-m.de>
Date: Sun, 4 Oct 2026 18:34:01 +0200
Subject: [PATCH 134/138] uhttpd: update to Git HEAD (2026-08-24)

82b4c79ec266 client: set TCP_NODELAY on accepted sockets
3aa7e1c76782 file: enforce Basic Auth realms for URL-prefix handlers

Link: https://github.com/openwrt/uhttpd/security/advisories/GHSA-5cgm-8h9x-v28c`
Link: https://github.com/openwrt/openwrt/pull/25613
(cherry picked from commit 26410f4160c9bf059aaef275cfba949df5cccb46)
Link: https://github.com/openwrt/openwrt/pull/25654
Signed-off-by: Hauke Mehrtens <hauke@hauke-m.de>
---
 package/network/services/uhttpd/Makefile | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/package/network/services/uhttpd/Makefile b/package/network/services/uhttpd/Makefile
index 8a2e5a6d5a..c2a32573c0 100644
--- a/package/network/services/uhttpd/Makefile
+++ b/package/network/services/uhttpd/Makefile
@@ -9,9 +9,9 @@ PKG_RELEASE:=1
 
 PKG_SOURCE_PROTO:=git
 PKG_SOURCE_URL=$(PROJECT_GIT)/project/uhttpd.git
-PKG_SOURCE_DATE:=2026-08-03
-PKG_SOURCE_VERSION:=60f64bec40c8113cf09815ec377761b1f4f95f22
-PKG_MIRROR_HASH:=702b8df1ba868db13ae93b28ddaa9c421bcbfa34e8633cae984029e7b50ec59f
+PKG_SOURCE_DATE:=2026-08-24
+PKG_SOURCE_VERSION:=373145f72c884c36a2b16f7f47e74ffae06bd754
+PKG_MIRROR_HASH:=3d1533281f0c78fc6f1118dd0635b7bd2d0c823e6c7ad63aec818769fac3cde9
 PKG_MAINTAINER:=Felix Fietkau <nbd@nbd.name>
 PKG_LICENSE:=ISC
 
-- 
2.34.1

